test-cases/gutenberg/inserter.md

```diff
@@ -0,0 +1,11 @@
+# Block Inserter - Test Cases
+
+##### TC0001
+
+**Blocks Are Sorted Properly In The Inserter**
```

Comment:
I see that on the rest of the test cases we don't use Start case (the first letter of each word capitalized) but Sentence case, what do you think about updating it?

Reply:
👍  Done!